consoleme/lib/aws.py

```diff
@@ -1811,7 +1811,7 @@ def allowed_to_sync_role(
     allowed = True
     allowed_tags = config.get("roles.allowed_tags", {})
     allowed_arns = config.get("roles.allowed_arns", [])
-    if not (allowed_tags or allowed_arns):
+    if not (allowed_tags and allowed_arns):
```

Comment:
I think you want to do: `if not allowed_tags and not allowed_arns:` here, since the way it is currently, it will return early if even one of them is undefined. Since `if not (allowed_tags and allowed_arns)` is logically equivalent to `if not allowed_tags or not allowed_arns`

Reply:
oof, good point